drivers/SmartThings/matter-switch/src/test/test_matter_switch_device_types.lua

```diff
@@ -394,6 +460,77 @@ local function test_init_parent_child_unsupported_device_type()
   })
 end
 
+local child_profiles = {
+  [child1_ep] = t_utils.get_profile_definition("light-level.yml"),
+  [child2_ep] = t_utils.get_profile_definition("light-color-level.yml"),
+  [child3_ep] = t_utils.get_profile_definition("light-color-level.yml")
+}
+
+local mock_children = {}
+for i, endpoint in ipairs(mock_device_parent_child_endpoints_out_of_order.endpoints) do
+  if endpoint.endpoint_id ~= parent_ep and endpoint.endpoint_id ~= 0 then
+    local child_data = {
+      profile = child_profiles[endpoint.endpoint_id],
+      device_network_id = string.format("%s:%d", mock_device_parent_child_endpoints_out_of_order.id, endpoint.endpoint_id),
+      parent_device_id = mock_device_parent_child_endpoints_out_of_order.id,
+      parent_assigned_child_key = string.format("%d", endpoint.endpoint_id)
+    }
+    mock_children[endpoint.endpoint_id] = test.mock_device.build_test_child_device(child_data)
+  end
+end
```

Comment:
this isn't in a function? What are you trying to do here out of curiosity?

Reply:
Oh I see, these are the mock children for the test. Should these be put in a function for consistency?

Reply:
perhaps this can be directly placed in this function `test_init_parent_child_endpoints_out_of_order` for clarity?

Reply:
Sure thing, updated as suggested in the latest commit

Reply:
Good call adding this test case Nick!

For the sake of consistency, I would move this test case to `test_multi_switch_parent_child_lights.lua` and just create new variable names for the `mock_children` and `child_profiles` variables to something like `mock_children_out_of_order` just so it doesn't conflict with the other existing variables in that file. Otherwise you could be able to keep the the same code you have here and just copy it over to that file.

These values will need to remove outside of the init function so they can be accessed in the test case 👍

Reply:
Moved as suggested!